src/ServiceControl/ConfigTransportConfig.cs

```diff
@@ -9,7 +9,7 @@ public TransportConfig GetConfiguration()
         {
             return new TransportConfig
             {
-
+                MaximumMessageThroughputPerSecond = 350,
```

Comment:
Do we need to expose this as a setting in app.config since this value might be different depending on hardware?